pkg/apis/admissionregistration/validation/validation.go

```diff
@@ -214,13 +216,18 @@ func validateAdmissionReviewVersions(versions []string, requireRecognizedAdmissi
 
 // ValidateValidatingWebhookConfiguration validates a webhook before creation.
 func ValidateValidatingWebhookConfiguration(e *admissionregistration.ValidatingWebhookConfiguration) field.ErrorList {
+	strictCost := false
+	if utilfeature.DefaultFeatureGate.Enabled(features.StrictCostEnforcementForWebhooks) {
+		strictCost = true
+	}
 	return validateValidatingWebhookConfiguration(e, validationOptions{
 		ignoreMatchConditions:                   false,
 		allowParamsInMatchConditions:            false,
 		requireNoSideEffects:                    true,
 		requireRecognizedAdmissionReviewVersion: true,
 		requireUniqueWebhookNames:               true,
 		allowInvalidLabelValueInSelector:        false,
+		strictCostEnforcement:                   strictCost,
```

Comment:
```suggestion
		strictCostEnforcement:                   utilfeature.DefaultFeatureGate.Enabled(features.StrictCostEnforcementForWebhooks),
```

(Just a suggestion to reduce verbosity)